src/TypeScript/Analyzer/EslintBridgeAnalyzer.cs

```diff
@@ -74,18 +77,17 @@ public async Task<IReadOnlyCollection<IAnalysisIssue>> Analyze(string filePath,
             await EnsureEslintBridgeClientIsInitialized(rulesProvider.GetActiveRulesConfiguration(), cancellationToken);
             var analysisResponse = await eslintBridgeClient.Analyze(filePath, tsConfig, cancellationToken);
 
+            if (analysisResponse.ParsingError != null && 
+                analysisResponse.ParsingError.Message.Contains(LinterIsNotInitializedError))
+            {
+                await eslintBridgeClient.InitLinter(rulesProvider.GetActiveRulesConfiguration(), cancellationToken);
```

Comment:
`rulesProvider.GetActiveRulesConfiguration()` was called above. Do we want to recalculate the value, or reuse the one from above?

Reply:
Good question. From this class' point of view, it shouldn't matter, but since we know that it's re-reading the file behind the scene, it'd be better to re-use the values. I'll change.